origins_workflow/origins_workflow.module

```diff
@@ -92,8 +92,8 @@ function origins_workflow_form_alter(&$form, FormStateInterface $form_state, $fo
  * Implements hook_menu_links_discovered_alter().
  */
 function origins_workflow_menu_links_discovered_alter(&$links) {
-  // Add a 'Site Themes' option in the dashboard menu
-  // if the nidirect_site_themes module is installed.
+  // Add  'Site Themes', 'Scheduled' and 'GPs' options to the
```

Comment:
Super minor observation: there's a double space in `Add  'Site Themes'` :)

Reply:
Fair enough, those things annoy me too. Now that you have pointed it out I can't leave it - so fixed now!